BUILD.md

```diff
@@ -36,7 +36,7 @@ See [System Requirements](https://xrpl.org/system-requirements.html).
 Building rippled generally requires git, Python, Conan, CMake, and a C++ compiler. Some guidance on setting up such a [C++ development environment can be found here](./docs/build/environment.md).
 
 - [Python 3.7](https://www.python.org/downloads/)
-- [Conan 1.55](https://conan.io/downloads.html)
+- [Conan 1.60](https://conan.io/downloads.html)
```

Comment:
If you're making this change, but still not recommending >=2.0, maybe add a note to that effect here.